Mathlib/Order/Hom/Basic.lean

```diff
@@ -1104,6 +1104,17 @@ end StrictMono
 protected def OrderIso.dual [LE α] [LE β] (f : α ≃o β) : αᵒᵈ ≃o βᵒᵈ :=
   ⟨f.toEquiv, f.map_rel_iff⟩
 
+section
+variable [LE α] [LE β] (f : α ≃o β)
```

Comment:
Why not include `OrderIso.dual` in this section?

Reply:
I think it's preferred for definitions to have variables not extracted.

Reply:
As in https://github.com/leanprover-community/mathlib4/pull/31638/ I agree with @tb65536 unless there is some unforeseen issue with this.